CHANGELOG.rst

```diff
@@ -11,6 +11,28 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
 
+[2.0.0] - 2020-11-05
+~~~~~~~~~~~~~~~~~~~~
+
+* Deprecate namespaced waffle classes: the WaffleFlag and WaffleSwitch classes exposed in `toggles` are now the namespace-less classes which were previously in `toggles.__future__`.
+    * If you were importing from `edx_toggles.toggles.__future__` before, then you simply need to import from `edx_toggles.toggles`. Importing from `__future__` will continue to work but will trigger a deprecation warning.
+    * If you were importing from `edx_toggles.toggles`, then you either need to:
+        * Migrate your legacy namespaced classes to the new-style classes (see the new behaviour below).
+        * Import `LegacyWaffleFlag` instead of `WaffleFlag` and `LegacyWaffleSwitch` instead of `WaffleSwitch`.
+    * The new classes introduce the following changes:
+        * The `WaffleSwitchNamespace._namespaced_name` and `WaffleFlagNamespace._namespaced_name` methods are replaced by the `WaffleSwitch.name` and `WaffleFlag.name` attributes.
+        * The `WaffleSwitchNamespace.is_enabled` method is replaced by the `WaffleSwitch.is_enabled` method.
+        * The `WaffleSwitchNamespace.set_request_cache_with_short_name` method is deprecated; instead, you could (but really shouldn't) use the `WaffleSwitch._cached_switches` property.
+        * The `WaffleSwitch.switch_name` attribute is deprecated: switches should only ever be referred to using their fully namespaced names.
+        * The `WaffleSwitch.namespaced_switch_name` attribute is replaced by `WaffleSwitch.name`.
+        * The `WaffleFlagNamespace.is_flag_active` method is replaced by `WaffleFlag.is_enabled`.
+        * The `WaffleFlagNamespace._monitor_value` method is replaced by `WaffleFlag.set_monitor_value`.
+        * The `WaffleFlagNamespace._cached_flags` attribute is replaced by the `WaffleFlag.cached_flags` method.
+        * The `WaffleFlag` and `WaffleSwitch` `module_name` constructor argument is now mandatory.
+        * The `WaffleFlag.flag_name` attribute is deprecated.
+        * The `WaffleFlag.namespaced_flag_name` attribute is replaced by `WaffleFlag.name`.
+        * The `WaffleFlag.waffle_namespace` attribute is deprecated.
```

Comment:
Again, instead of "deprecated" maybe something like:
```
      * The `WaffleFlag.waffle_namespace` attribute no longer exists, since there is no longer a separate namespace object.
```